EarlGrey/Common/GREYTestHelper.h

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+/**
+ *  Provides interface for test helper methods.
+ */
+@interface GREYTestHelper : NSObject
+
+/**
+ *  Enables fast animation.
```

Comment:
A bit more context on when and where it should be used would be helpful for others trying to decide whether to use this method.